symplyphysics/definitions/wavelength_is_velocity_by_frequency.py

```diff
@@ -0,0 +1,37 @@
+from sympy import pi
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+## The main characteristics of any wave are it's frequency, wavelength, magnitude and spreading velocity.
+## Magnitude is power characteristic of wave.
+## Other characteristics are dependent from each over.
+
+## Definition: λ = V / (n * f)
+## Where: λ is wavelength,
+## V is spreading velocity,
+## n is refraction factor of the media,
+## f is frequency.
+
+wavelength, spreading_velocity, refraction_factor, wave_frequency = symbols('wavelength, spreading_velocity, refraction_factor, wave_frequency')
+
+definition = Eq(wavelength, spreading_velocity / (refraction_factor * wave_frequency))
```

Comment:
I doubt it is a definition of wavelength but we can keep it this way.

Reply:
Actually, let's derive it from 2 more basic laws. First is definition of refraction coefficient and second is definition of wavelength from frequency.